src/components/examples-tools/example-controls/example-controls.scss

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
I think we really only need the event printer right now, so I'm going to pull that out to a separate, much smaller, PR.

Reply:
The event printer has been moved to #117

Reply:
Ah, no, it's actually example-value we are using in lime-crm-building-blocks. I'll split that to a separate one too, so the separate parts don't have to be held up by discussions about other parts.

Reply:
example-value moved to #118, but I saw now that the docs for that component seem copy-pasted from event-printer, so they need updating.